.github/workflows/codeforafrica-build-deploy.yml

```diff
@@ -0,0 +1,87 @@
+name: codeforafrica.org | Build and Deploy
+on:
+  push:
+    branches: [main]
+
+env:
+  DOKKU_REMOTE_BRANCH: "master"
+  DOKKU_REMOTE_URL: "ssh://dokku@ui-1.prod.codeforafrica.org/code-for-africa-ui"
+  IMAGE_NAME: "codeforafrica/codeforafrica-ui"
```

Comment:
Match image and Dokku app name if possible. If we're going with `codeforafrica-ui`, then both should be that. Likewise if we're going with `code-for-africa-ui` or just `codeforafrica`.